python/src/equistore/operations/unsafe.py

```diff
@@ -0,0 +1,136 @@
+"""
+Disabling Checks
+================
+
+.. warning::
+
+    **Using equistore without checks is unsafe and can lead to unwanted results and may
+    produce cryptic error messages. Therefore, disabling checks are recommended for
+    nobody and should only be used by advanced users!**
+
+Equistore performs `shape` as well as `consistency` checks for the metadata during each
+operation. These functions allow to disable these checks. Checks can either be disabled
+temporarily via compound statements (see below) or globally and permanently by calling
+:py:class:`equistore.disable_checks()`
+
+You can also control the checks via the environment variable `EQUISTORE_DISABLE_CHECKS`
+To disable checks set
+
+.. code-block:: bash
+
+    export EQUISTORE_DISABLE_CHECKS=1
+
+
+Note that using :py:func:`equistore.enable_checks()` or
+:py:class:`equistore.disable_checks()` overwrite the defintion of the enviroment
+variable.
+
+Controlling methods
+-------------------
+
+.. autofunction:: equistore.checks_enabled
+
+.. autoclass:: equistore.enable_checks
+
+.. autoclass:: equistore.disable_checks
+
+"""
+
+from equistore.utils import _parse_bool_env_var
```

Comment:
Since this is not used anywhere else, should we define this function in this module?

Reply:
Yes, I don't mind where this lives for now. I just thought it might be useful in the future.